packages/gridstack/src/components/SaveDashboardModal/SaveDashboardModal.vue

```diff
@@ -0,0 +1,53 @@
+<template>
+  <div class="save-dashboard-modal">
+    <h3 class="header1">Konfigurationsname eingeben</h3>
+    <div class="header2">
+      Bitte wählen Sie den Dashboard-Konfigurationsnamen
+    </div>
+    <BaseInputV2
+      v-model="name"
+      label="Name"
+      :validation-rules="required"
+      name="name"
+      class="base-input"
+    />
+    <BaseButton :disabled="!name" @click="submitAndClose" class="save-button">
+      Speichern
+    </BaseButton>
+  </div>
+</template>
+
+<script lang="ts" setup>
+import { ref, computed } from 'vue'
+import { BaseInputV2, BaseButton } from '@lion5/component-library'
+
+const name = ref('')
+// TODO: this seems not right. The computed value is applied to the validation-rules prop of the BaseInputV2 component but the default value 'Erforderlich' seems not to be a validation rule.
+const required = computed(() => ({
+  required: (value) => !!value || 'Erforderlich'
+}))
```

Comment:
See TODO

Reply:
I thought it was the error message. That it can either be boolean or string which implies false and passes the error message. 

But removed the 'Erforderlich' anyways as we keep the button disabled until the input has text.

Reply:
Hmm. That can be the case. I did not find something like that in the veeValidate documentation. But maybe the function is used for the error message.